SPECS/grub2/grub2.spec

```diff
@@ -339,6 +349,22 @@ cp $GRUB_PXE_MODULE_SOURCE $EFI_BOOT_DIR/$GRUB_PXE_MODULE_NAME
 %{_sysconfdir}/sysconfig/grub
 %attr(0644,root,root) %ghost %config(noreplace) %{_sysconfdir}/default/grub
 %ghost %config(noreplace) /boot/%{name}/grub.cfg
```

Comment:
shouldn't both these configuration files move over to the `grub2-configuration` package?

Reply:
done.